api/src/champion-mastery/controller.ts

```diff
@@ -0,0 +1,8 @@
+import { API } from 'lambda-api';
+import { getChampionMastery } from './service';
+
+export const championMasteryApi = (api: API) => {
+  api.get('/', async () => {
+    return getChampionMastery();
+  });
```

Comment:
Was there a reason we used anonymous functions vs normal ones? I think I might prefer normal ones, only because for the UI, it's easier to see them in the debugger. And so I'd wanna use named ones here to be consistent with the UI

Reply:
No reason. I'll switch to normal